configserver/src/main/java/com/yahoo/vespa/config/server/tenant/ContainerEndpointsCache.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.config.server.tenant;
+
+import com.yahoo.config.provision.ApplicationId;
+import com.yahoo.path.Path;
+import com.yahoo.vespa.config.SlimeUtils;
+import com.yahoo.vespa.curator.Curator;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.List;
+
+
+/**
+ * Persists assignment of rotations to an application to ZooKeeper.
+ * The entries are RotationAssignments, which keep track of the container
```

Comment:
s/RotationAssignments/ContainerEndpoint/. If you use `{@link class}`, I think IntelliJ refactors this automatically.